toolkits/landcover/impl/dataset.js

```diff
@@ -153,33 +168,34 @@ Dataset.prototype.computeCommonBandNames_ = function(bandNames) {
  * @return {!Dataset}
  */
 Dataset.prototype.addBandIndices = function(var_names) {
+  var dataset = this.clone_();
   var indices = Array.prototype.slice.call(arguments);
 
   // Compute the transformation to common band names.
-  var bandNames = this.collection_.first().bandNames();
+  var bandNames = dataset.collection_.first().bandNames();
```

Comment:
There's an unclean mixture of uses of this and dataset in here.
Once you've cloned, you shouldn't use this any more, so either
use dataset throughout or don't clone until you're about to modify state (#185)

Reply:
The two instances are identical at this point, so I'd say the one reference to `this` in the following line is inconsistent, but "unclean" is a bit of a stretch. I'll update to make them consistent.